chore: polish kb63-72 files (#1012)
keymanapp/help.keyman.com: knowledge-base/kb0063.md

- When using right-to-left languages, the selected Windows language determines the direction of the text. You should always associate a right-to-left Windows language with a Keyman keyboard layout that requires right-to-left support.
- Some applications use the selected Windows language to determine how text should be displayed. You may find this to be particularly the case for some Indian languages.
- Many applications use the selected Windows language to determine which spell checking or other language tools should be applied to a text. Associating the correct Windows language with you Keyman keyboard layout will ensure that the correct language tools are used on the text.

The good news is that most Keyman keyboard layouts are now designed to automatically configure associations with a Windows language and layout. This means, in many cases, you should not need to configure or associate anything. When you do need to associate Keyman keyboards for best performance, follow the steps below.

## Associating in Windows XP
Installing a Windows Language and Windows Layout

> - Open the Control Panel.
> - Select Regional and Language Options.
> - Select the Languages tab.
> - Click on the Details button.
> - Click on the Add button.
> - Select the language to be installed (from the input language list). This language should be the language of your Keyman keyboard. For example, Greek.
> - Select the **US** keyboard layout from the "Keyboard layout/IME" list. This will provide best peformance with <ins>most</ins> keyboard layouts. **Note**: the Windows English (USA, International) keyboard is not identical to the Windows US keyboard. We **do not** recommend selecting the English (USA, International) keyboard.
> - Click OK, OK, OK.

Author: ```suggestion
1. Open the Control Panel.
2. Select Regional and Language Options.
3. Select the Languages tab.
4. Click on the Details button.
5. Click on the Add button.
6. Select the language to be installed (from the input language list). This language should be the language of your Keyman keyboard. For example, Greek.
7. Select the **US** keyboard layout from the "Keyboard layout/IME" list. This will provide the best performance with <ins>most</ins> keyboard layouts. **Note**: the Windows English (USA, International) keyboard is not identical to the Windows US keyboard. We **do not** recommend selecting the English (USA, International) keyboard.
8. Click OK, OK, OK.
```

Configuring Keyman Desktop

> - Start Keyman Configuration (from the Keyman menu in the taskbar).
> - Click the Languages tab.
> - Find your language in the Installed language list.
> - Select your Keyman keyboard layout from the dropdown menu.
> - Click OK.

Author: ```suggestion
1. Start Keyman Configuration (from the Keyman menu in the taskbar).
2. Click the Languages tab.
3. Find your language in the Installed language list.
4. Select your Keyman keyboard layout from the dropdown menu.
5. Click OK.
```

## Associating in Windows Vista & Windows 7
Installing a Windows Language and Windows Layout

> - Open the Control Panel.
> - Under Clock, Language and Region, select "Change keyboards or other input methods". **Note**: in Classic View, select Regional and Language Options, and the tab "Keyboards and Languages."
> - Click Change Keyboards.
> - Click Add.
> - Select the language to be installed (from the input language list). This language should be the language of your Keyman keyboard. For example, Greek.
> - Select the **US** keyboard layout from the "Keyboard layout/IME" list. This will provide best peformance with <ins>most</ins> keyboard layouts. **Note**: the Windows English (USA, International) keyboard is not identical to the Windows US keyboard. We **do not** recommend selecting the English (USA, International) keyboard.
> - Click OK, OK, OK.

Author: ```suggestion
1. Open the Control Panel.
2. Under Clock, Language and Region, select "Change keyboards or other input methods". **Note**: in Classic View, select Regional and Language Options, and the tab "Keyboards and Languages."
3. Click Change Keyboards.
4. Click Add.
5. Select the language to be installed (from the input language list). This language should be the language of your Keyman keyboard. For example, Greek.
6. Select the **US** keyboard layout from the "Keyboard layout/IME" list. This will provide best peformance with <ins>most</ins> keyboard layouts. **Note**: the Windows English (USA, International) keyboard is not identical to the Windows US keyboard. We **do not** recommend selecting the English (USA, International) keyboard.
7. Click OK, OK, OK.
```

Configuring Keyman Desktop

> - Start Keyman Configuration (from the Keyman menu in the taskbar).
> - Click the Languages tab.
> - Find your language in the Installed language list.
> - Select your Keyman keyboard layout from the dropdown menu.
> - Click OK.

Author: ```suggestion
1. Start Keyman Configuration (from the Keyman menu in the taskbar).
2. Click the Languages tab.
3. Find your language in the Installed language list.
4. Select your Keyman keyboard layout from the dropdown menu.
5. Click OK.
```
